CONTRIBUTING.md

```diff
@@ -95,8 +95,6 @@ Class SomeClass
     // Console
     public void PrintMessage(string p_Message) {...}
     
-    //-----------------------------------------------------
```

Comment:
I don't like such separators so i removed them.
If you want we can discuss this further …

Reply:
No, it is ok